### ISSUE TITLE
disable too-many-positional-arguments causing erroneous messages in Python 3.11

### ISSUE THREAD

**Issue by 1John419** [PyCQA/pylint#7915]
### Bug description

Just switched to Python 3.11. Now pylint  is reporting errors in all my python source, which it was not doing using Python 3.10.

Discovered the problem does not occur if disable=too-many-positional-arguments is commented out in my .pylintrc file.

Even a python file containing only a shebang (or a completely empty file) produces the error.

### Command used

```shell
Run from Visual Studio Code 1.74.
```

### Pylint output

```shell
Unknown option value for '--disable', expected a valid pylint message and got 'too-many-positional-arguments'
```

### Expected behavior

With Python 3.10, there were no erroneous messages.

### Pylint version

```shell
pylint 2.15.0
astroid 2.12.13
Python 3.11.0 (main, Oct 24 2022, 18:26:48) [MSC v.1933 64 bit (AMD64)]
```

### OS / Environment

Windows 10 Pro 64-bit (22H2)

**Comment by Pierre-Sassoulas** [comment]
I don't think it's python 3.10 vs 3.11. I suppose you also upgraded from pylint < 2.14 to 2.15.0 at the same time because ``too-many-positional-arguments`` is not a pylint message and never was (There's https://pylint.pycqa.org/en/latest/user_guide/messages/error/too-many-function-args.html though), The parsing of config got better in 2.14 and now pylint warn you about unknown value in disable/enable. You should remove ``too-many-positional-arguments``  from your conf.
